app-shared-tests/build.gradle

```diff
@@ -60,52 +60,35 @@ android {
 dependencies {
     implementation(project(":app"))
 
-    // koin
-    api "io.insert-koin:koin-core:$koin_version"
-    api "io.insert-koin:koin-test:$koin_version"
-    api "io.insert-koin:koin-test-junit4:$koin_version"
-    api "io.insert-koin:koin-android:$koin_version"
-    api "io.insert-koin:koin-android-test:$koin_version"
-    api "io.insert-koin:koin-androidx-compose:$koin_version"
+    // Koin
+    api(libs.bundles.koin)
 
-    // assertion library
-    // cannot use 0.34.0 due to an existing bug
+    // Assertion Library
+    // Cannot use 0.34.0 due to an existing bug.
     // https://github.com/robfletcher/strikt/issues/259
```

Comment:
**suggestion**: Right now I think this comment would be better suit to version catalog